diffkemp/building/cc_wrapper_gen.py

```diff
@@ -0,0 +1,23 @@
+from subprocess import check_call
+import os
+
+
+def generate_cc_wrapper(rpython='rpython', builddir='build'):
```

Comment:
We're using double quotes everywhere, could you follow that?

Reply:
Actually `grep -r "'"` shows that we are not always using double quotes, a few lines in `diffkemp/cli.py` and `diffkemp/syndiff/function_syntax_diff.py` use single quotes, that is probably not intentional though. My use of single quotes wasn't intentional either, I guess I was influenced by an RPython code example I used for `cc_wrapper_template.py`.

Either way I'll change the occurences of single quotes in this patch to double quotes, the rest can be dealt with later if the affected code is changed.

Reply:
Yes, agreed.